x-pack/plugins/security_solution/common/endpoint/types.ts

```diff
@@ -41,14 +41,30 @@ type ImmutableMap<K, V> = ReadonlyMap<Immutable<K>, Immutable<V>>;
 type ImmutableSet<T> = ReadonlySet<Immutable<T>>;
 type ImmutableObject<T> = { readonly [K in keyof T]: Immutable<T[K]> };
 
+export interface EventStats {
+  /**
+   * The total number of related events (all events except process and alerts) that exist for a node.
+   */
+  total: number;
+  /**
+   * A mapping of ECS event.category to the number of related events are marked with that category
+   * For example:
+   *  {
+   *    network: 5,
+   *    file: 2
+   *  }
+   */
+  categories: Record<string, number>;
```

Comment:
Will the UI need to convert this to a map? If so, maybe this type would be better:
```
categories: Map<string, number>
```
or if an ecs.category type or enum exists:
```
categories: Map<ECSCategory, number>
```

Reply:
It's `Record<string, number>` in the UI types ( resolver/types.ts : 159 )